tests/models/musicgen/test_modeling_musicgen.py

```diff
@@ -124,6 +120,7 @@ def __init__(
         pad_token_id=99,
         bos_token_id=99,
         num_codebooks=4,
+        audio_channels=1,
```

Comment:
`audio_channels=1` is the default in the config. In other words, this doesn't change the tests, but allows us to quickly override the config (see overloaded test below)